@navikt/ds-tokens/src/index.js

```diff
@@ -1,24 +1,39 @@
 const Color = require("color");
 
+// https://github.com/hihayk/scale/blob/69b766bba2db046d3e8cb4026ae32a32c897f9ff/src/utils.js#L44
+const mixColors = (color, step, amount, mixColor) => {
+  const saturation = Math.round(Color(color).hsl().color[1]);
+  // Setting directly to hex returns different result...
+  return Color(
+    Color(color)
+      .saturate(((step + 1) / 5) * (saturation / 100))
+      .mix(Color(mixColor), ((amount / 100) * (step + 1)) / 5)
+      .string()
+  ).hex();
+};
+
+const lightColor = (color, step) => mixColors(color, step, 100, "white");
+const darkColor = (color, step) => mixColors(color, step, 85, "black");
+
 const baseColors = {
   blue: "#0067c5",
   deepblue: "#005B82",
   lightblue: "#66CBEC",
   orange: "#FF9100",
   green: "#06893A",
   red: "#BA3A26",
+  purple: "#634689",
+  limegreen: "#a2ad00",
 };
+
 const white = "#ffffff";
-const darkgray = "#3e3832";
+const darkgray = "#262626";
```

Comment:
Ser at `#262626` er definert to steder i denne fila nå (navds-color-gray-90). Er det en ryddig måte å unngå det på? Kanskje gråfargene kan defineres på en mer logisk måte nå? F.eks. `Color("black").lighten()` eller no.

Reply:
Fant ingen god løsning med .lighten(), men tenker dette er en ok løsning 
```shell
90: { value: "{navds.color.darkgray.value}" },
```

Reply:
Jeg hørte med Wei: Han sa gråfargene var justert for hånd for å sørge for god kontrast, så da gir det mening at det ikke er noen clean måte å generere de på.

Reply:
Jeg kom til å tenke på en ting: Trenger vi egentlig en token som heter darkgray? Betyr ikke `gray-90` "darkgray"?

Reply:
Ser for meg det kan være greit å ha `gray-90` som en egen verdi siden den har en spesifikk bruk, men hvis vi går gjennom å spesifiserer tokensene bedre er det kanskje ikke grunn til det lengre. Typ color-text, color-shadow